tests/test_models/test_autoarima_model.py

```diff
@@ -144,3 +145,32 @@ def test_forecast_1_point(example_tsds):
 def test_save_load(example_tsds):
     model = AutoARIMAModel()
     assert_model_equals_loaded_original(model=model, ts=example_tsds, transforms=[], horizon=3)
+
+
+@pytest.mark.parametrize(
```

Comment:
We actually did't touch this adapter, I guess it is not really necessary. If you thing that it is useful also add test for components sum

Reply:
Yes, we can remove this test